dist/robots.cfg

```diff
@@ -108,6 +112,7 @@ RobotUA <<EOR
     panscient.com,
     ParaSite,
     pavuk,
+	Pinterestbot,
```

Comment:
@perusiongreg Looks like there's different indentation for Pinterestbot. Can you make it indent like the surrounding lines?

Reply:
fixed that. was tab vs spaces

Reply:
Regarding the Outlook, I think it pertains more to the Preview functions that query the site when an email is viewed within the application .. that preview uses the Outlook UA, same as BingPreview. I don't have Outlook to test against, so cannot say for sure, but this is on BS which is a high traffic sight and very "picky" and outspoken customers.  So at least we have good odds in our favor.